modules/apps/segments/segments-web/test/js/utils/odata.es.js

```diff
@@ -58,187 +72,333 @@ describe('odata-util', () => {
 				"((((firstName eq 'test' or firstName eq 'test') and firstName eq 'test') or firstName eq 'test') and firstName eq 'test')"
 			);
 		});
-	});
 
-	describe('translateQueryToCriteria', () => {
-		it('translates a query string into a criteria map', () => {
-			expect(
-				ODataUtil.translateQueryToCriteria("(firstName eq 'test')")
-			).toEqual({
+		it('is able to translate a query string to map and back to string', () => {
```

Comment:
(non-blocking feedback)

You can make some of these shorter and therefore easier to fit on one line by dropping the "is able to". eg.

```js
it('translates a query string to map and back', () => {
```

Reply:
Great @wincent, thank you so much for the feedback. I'll change it.